contents/handbook/people/team-structure/platform.md

```diff
@@ -40,34 +39,28 @@ Concrete things we take responsibility over:
 ## Customer
 
 - Other Small Teams in making sure they have the tools (databases, queues, etc) and the ability to deploy effortlessly that they need to build
-- End users (Both cloud and on-prem teams)
+- Self deployed teams
+- app.posthog.com users
 
-## Output metrics
+## Metrics
 
-### VPC
+### Self Deploy 
 ###### Retention 
 - Metric: Retention
 - Objective: Better than cloud
+###### Ease of deploy 
+- Metric: NPS on survey of how install experience was
+- Objective: NPS of > 8
 ### Cloud
 ###### Data Loss
-- Metric: Data loss %
-- Objective: < 0.1%
-###### Uptime 
+- Metric: % of events dropped 
+- Objective: < 0.01%
+###### Uptime SLO 
 - Metric: Uptime
 - Objective: > 99.99%
```

Comment:
I liked @guidoiaquinti 's proposal here to change it to 99% in https://github.com/PostHog/company-internal/issues/278